templates/bundles/FOSUserBundle/Security/login_content.html.twig

```diff
@@ -1,59 +1,108 @@
 {% trans_default_domain 'FOSUserBundle' %}
 
-<div class="container">
-  <div class="row" id="pwd-container">
-    <div class="col-md-4 offset-md-4 col align-self-center">
-      <section class="login-form">
-
-          {% if is_granted("IS_AUTHENTICATED_REMEMBERED") %}
-          <form action="{{ path("fos_user_security_check") }}" method="post" role="login">
-              <img src="{{ cdn_dns ~ '/' ~ bucket_name ~ '/' ~ app.user.getProfileImage()}}" alt="profile image" class="rounded-circle" style="width: 100px;">
-              <p class="text-center">
-                  {{ 'layout.logged_in_as'|trans({'%username%': app.user.username}, 'FOSUserBundle') }}
-              </p>
-              <a href="{{ path('app_profile', {'slug': app.user.slug}) }}" class="btn btn-success btn-block">
-                  <i class="fa fa-eye"></i>
-                  visualizar meu perfil
-              </a>
-              <a href="{{ path('fos_user_profile_edit') }}" class="btn btn-primary btn-block">
-                  <i class="fa fa-edit"></i>
-                  editar meu perfil
-              </a>
-              <a href="{{ path('fos_user_security_logout') }}" class="btn btn-danger btn-block">
-                  <i class="fa fa-close"></i>
-                  {{ 'layout.logout'|trans({}, 'FOSUserBundle') }}
-              </a>
-          </form>
-          {% else %}
-              <form action="{{ path("fos_user_security_check") }}" method="post" role="login">
-
-                    <h3 class="text-center"><b>My</b> Profile</h3><br/>
-
-                    {% if csrf_token %}
-                        <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
-                    {% endif %}
-
-                    <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" placeholder="{{ 'security.login.username'|trans }}" class="form-control" />
-
-                    <input type="password" name="_password" class="form-control input-lg" id="password" placeholder="{{ 'security.login.password'|trans }}" required="required" />
-
-                    <input type="checkbox" id="remember_me" name="_remember_me" value="on" />
-                    <p for="remember_me">{{ 'security.login.remember_me'|trans }}</p>
-
-                    <div class="pwstrength_viewport_progress"></div>
-
-                    {% if error %}
-                        <div>{{ error.messageKey|trans(error.messageData, 'security') }}</div>
-                    {% endif %}
-
-                    <button type="submit" id="_submit" name="_submit" class="btn btn-lg btn-primary btn-block">{{ 'security.login.submit'|trans }}</button>
-                    <div>
-                        <a href="{{path('fos_user_registration_register')}}">{{'layout.register'| trans}}</a> or <a href="{{path('fos_user_resetting_request')}}">{{'resetting.email.subject' | trans}}</a>
+{{ encore_entry_link_tags('web-site') }}
+
+<section class="section">
```

Comment:
![Screenshot from 2020-10-06 11-03-54](https://user-images.githubusercontent.com/6358755/95181718-ee1fa300-07c3-11eb-88cc-ec0a1e76b199.png)


Hey @sourcefactory-at could you check when you put a wrong password, could you put this `invalid credentials` red

and I think this `register` and reset password should be side to side